packages/sketch/src/tools/sharedStyles.js

```diff
@@ -61,15 +61,28 @@ export function syncSharedStyle(
  * @param {Document} document
  * @param {string} name
  * @param {string} value
+ * @param {string} type
  * @returns {SharedStyle}
  */
-export function syncColorStyle(document, name, value) {
-  return syncSharedStyle(document, name, {
-    fills: [
-      {
-        color: value,
-        fillType: Style.FillType.Color,
-      },
-    ],
-  });
+export function syncColorStyle(document, name, value, type) {
```

Comment:
Instead of adding a `type` to the end of this and updating all the call signatures that exist already, would it make sense to add a `syncFillStyle` and `syncBorderStyle` helper and then we can call them separately?

Reply:
so would those two additional helpers be called by the sharedStyles helpers?